arangod/VocBase/LogicalCollection.h

```diff
@@ -232,6 +232,9 @@ class LogicalCollection: public LogicalDataSource {
   virtual void setStatus(TRI_vocbase_col_status_e);
 
   // SECTION: Serialisation
+  void toVelocyPack(velocypack::Builder&, bool translateCids,
```

Comment:
what is this one?

Reply:
@sleto-it No clue where this came from. Deleted these lines which don't exist in devel. Should be okay now.